src/graphql/fragments/colony.graphql

```diff
@@ -68,6 +68,9 @@ fragment Watcher on WatchedColonies {
 fragment WatchedColony on Colony {
   colonyAddress: id
   name
+  nativeToken {
```

Comment:
I was seeing some errors like this `Type 'WatchedColonyFragment | ColonyFragment' is not assignable to type 'ColonyFragment | undefined'.
  Property 'nativeToken' is missing in type 'WatchedColonyFragment' but required in type 'ColonyFragment'.`

Adding nativeToken to WatchedColony resolved these errors.

Reply:
@Julianahlk I think you were seeing this error because there are differences between `ColonyFragment` and `WatchedColonyFragment`. My guess is that we have these 2 independent fragments on `Colony` for a reason, so I think that making them the same by adding `nativeToken` kind of defeats the point of having 2 separate ones. Maybe you should look at the required types and adjust them to allow for a `WatchedColonyFragment` instead.